json_schemas/services-update.json

```diff
@@ -6,13 +6,9 @@
   "properties": {
     "updated_by": {
       "type": "string"
-    },
-    "update_reason": {
```

Comment:
`dmutils.apiclient` is still sending `update_reason`, so we should either keep it as non-required field or set `additionalProperties: true` for now